vendor/acquia/blt/config/build.yml

```diff
@@ -0,0 +1,275 @@
+  build-dependencies: true
+  dir: ${repo.root}/deploy
+  exclude_file: ${blt.root}/scripts/blt/deploy/deploy-exclude.txt
+  exclude_additions_file: ${repo.root}/blt/deploy-exclude-additions.txt
+  gitignore_file: ${blt.root}/scripts/blt/deploy/.gitignore
+
+# File and Directory locations.
+docroot: ${repo.root}/docroot
+
+# Drupal Account Credentials. These are used for installing Drupal.
+drupal:
+  #account.name: admin
+  account.mail: no-reply@acquia.com
+  locale: en
+  local_settings_file: ${docroot}/sites/${site}/settings/local.settings.php
+  settings_file: ${docroot}/sites/${site}/settings.php
+  db:
+    database: drupal
+    username: drupal
+    password: drupal
+    host: localhost
+    port: 3306
+
+drush:
+  bin: ${composer.bin}/drush
+  dir: ${docroot}
+  alias-dir: ${repo.root}/drush/sites
+  sanitize: true
+  ansi: true
+
+git:
+  # The value of a hook should be the file path to a directory containing an
+  # executable file named after the hook. Changing a hook value to 'false' will disable it.
+  # You should execute `blt blt:init:git-hooks` after modifying these values.
+  hooks:
+    pre-commit: ${blt.root}/scripts/git-hooks
+    commit-msg: ${blt.root}/scripts/git-hooks
+  commit-msg:
+    # Commit messages must conform to this pattern.
+    pattern: "/(^${project.prefix}-[0-9]+(: )[^ ].{15,}\\.)|(Merge branch (.)+)/"
+    # Human readable help description explaining the pattern/restrictions.
+    help_description: "The commit message should include your project prefix,
+                      followed by a hyphen and ticket number, followed by a colon and a space,
+                      fifteen characters or more describing the commit, and end with a period."
+    # Provide an example of a valid commit message.
+    example: "${project.prefix}-123: Update module configuration."
+  user:
+    # Name and email to use for the purposes of Git commits if you don't want to
+    # use global Git configuration.
+    name: ''
+    email: ''
+
+# You may provide a list of sites for BLT to run commands against,
+# otherwise BLT will generate this sites list based on directories
+# in ${docroot}/sites/*/
+#
+# multisites:
+#  - default
+
+# @todo Move to subkey of validate.
+phpcs:
+  standard: Drupal,DrupalPractice
+
+#$ @todo Move to subkey of fix.
+phpcbf:
+  filesets:
+    - files.php.custom.modules
+    - files.php.tests
+    - files.php.custom.themes
+    - files.frontend.custom.themes
+
+project:
+  local:
+    uri: ${project.local.protocol}://${project.local.hostname}
+
+setup:
+  # Valid values are install, sync, import.
+  strategy: install
+  # If setup.strategy is import, this file will be imported. File path is
+  # relative to drupal docroot directory.
+  dump-file: null
+
+sync:
+  # By default, files are not synced during sync:refresh.
+  # Set this value to 'true' or pass -D sync.files=true
+  # to override this behavior.
+  files: false
```

Comment:
I think this needs to be true too, I am going to try it.

Reply:
Ooh, good idea!